ios/FluentUI/Vnext/List/List.swift

```diff
@@ -69,19 +69,17 @@ public struct MSFListView: View {
             }
     }
 
-    private func dividers(_ sections: [MSFListSectionState]) -> [MSFListSectionState] {
-        var cells: [MSFListCellState]
-        var hasDividers: Bool
-        for sectionIndex in 0...sections.count - 1 {
-            cells = sections[sectionIndex].cells
-            hasDividers = sections[sectionIndex].hasDividers
-            for cellIndex in 0...cells.count - 1 {
-                if cellIndex != cells.count - 1 {
-                    cells[cellIndex].hasDivider = hasDividers
+    private func updateCellDividers() -> [MSFListSectionState] {
+        state.sections.forEach { section in
+            section.cells.forEach { cell in
+                if cell != section.cells.last {
+                    cell.hasDivider = false
```

Comment:
You can do this with only one ternary operator:

`cell.hasDivider = section.hasDividers && cell != section.cells.last`

Or to be more efficient and not check the condition all the time you can do:

```
cell.hasDivider = section.hasDividers // (inside the loops)

section.cells.last.hasDivider = false  // (outside of the loops)
```